### ISSUE TITLE
AltStore accessibility problems

### ISSUE THREAD

**Issue by devinprater** [rileytestut/AltStore#145]
AltStore is a great platform for apps, without Apple's restrictions. It could be a great resource for blind developers, who may not have the money to pay to be on Apple's store, or who may not want to be subject to Apple's rules.

Blind iOS users use the [VoiceOver](https://developer.apple.com/accessibility/ios/) screen reader to navigate and use their devices. Apps sometimes can be accessible without direct focus on accessibility in the app, and some parts of AltStore are okay, like the news tab. Other parts, like Browse and My Apps, could use a bit of work.

There are a few accessibility issues with the app, meaning that users will find it frustrating to download apps that are on the store, and deal with temporarily revoking apps to save slots.

* On the browse page, I cannot tell which apps are beta apps and which are not.
* When you press the "free" button, either in the app's page or on the main "browse" page, the button becomes unlabeled. I can't tell the progress of the download, or if it failed.
* When on the "my apps" page, after the "refresh button" there is an element that is empty to VoiceOver. Using its OCR technology, VoiceOver can gleam "expires in…" but that's all. I have attempted to download Provinence, and Delta, for example. After the unlabeled element, there is "5 app ID's remaining."
* In the "view App ID's" section of the "my apps" page, after the explanation text, there are many elements, which I assume are the apps which are installed, and VoiceOver detects Delta, but this is OCR, which is inferior to true app accessibility.

I recommend turning on VoiceOver, "Hey Siri, turn on VoiceOver," after reading its basic usage, and navigating your app with your eyes closed. This will give you some idea of what using VoiceOver is like, so that you may find even better ways of optimizing your apps for use by blind people, like using computer vision on Delta's game screen, and other ideas.

I hope this helps you to bring greater accessibility to your apps, and stretch the boundaries of iOS, and accessibility, even further.
